zap/src/main/java/org/zaproxy/zap/utils/ZapNumberSpinner.java

```diff
@@ -55,6 +59,19 @@ public ZapNumberSpinner(int minValue, int defaultValue, int maxValue) {
                 .getTextField()
                 .setFormatterFactory(
                         new DefaultFormatterFactory(new ZapNumberFormatter(minValue, maxValue)));
+        DefaultEditor editor = (DefaultEditor) getEditor();
+        editor.getTextField()
```

Comment:
Better create a variable with the above text field and use the same for the two tweaks.

Reply:
With above I was referring to `((NumberEditor) getEditor()).getTextField()`.